app/assets/javascripts/discourse/app/initializers/register-service-worker.js

```diff
@@ -1,10 +0,0 @@
-import { registerServiceWorker } from "discourse/lib/register-service-worker";
```

Comment:
This file is moved to `app/assets/javascripts/discourse/app/instance-initializers/register-service-worker.js` but git lost track of it